frontend/components/panels/EmployeePanel.js

```diff
@@ -165,42 +160,48 @@ class EmployeePanel extends React.Component {
     let linkElement = null;
     if (employee.url && !macros.isMobile) {
       linkElement = (
-        <span className={ css.link }>
-          <a key='jfdalsj' target='_blank' rel='noopener noreferrer' className={ css.inlineBlock } href={ employee.url }>
+        <span className='link'>
+
+          <a
+            key='jfdalsj'
+            target='_blank'
+            rel='noopener noreferrer'
+            className='inlineBlock'
+            href={ employee.url }
+          >
             <img src={ globe } alt='globe' />
           </a>
         </span>
       );
     }
 
     return (
-      <div className={ `${css.container} ui segment` }>
+      <div className='container ui segment'>
         <div
-          className={ css.header }
+          className='header'
           onClick={ titleClickListener }
           role='button'
           tabIndex={ 0 }
         >
           {chevron}
-          <span className={ css.titleText }>
+          <span className='titleText'>
             {employee.name}
           </span>
           {linkElement}
         </div>
 
-        <div className={ cx({
-          displayNone: !this.state.showMoreThanTitle && macros.isMobile,
-          body: true,
-        }) }
+        <div style={{
+          display: (!this.state.showMoreThanTitle && macros.isMobile) && 'none',
+          padding: 20,
+        }}
         >
-          <div className={ `${css.inlineBlock} ${css.contactBox}` }>
+          <div className='inlineBlock contactBox'>
             {this.constructor.injectBRs(firstColumn)}
           </div>
-          <div className={ cx({
-            inlineBlock: true,
-            mobileSecondColumn: macros.isMobile,
-            desktopSecondColumn: !macros.isMobile,
-          }) }
+          <div style={{
+            display: 'inline-block',
+            'padding-left': macros.isMobile ? 20 : 100,
+          }}
```

Comment:
we could use `@media only screen and (min-width: 767px)` in the scss to apply different styles to mobile and desktop here